### DISCUSSION TITLE
Welcome to Hacktoberfest_2024 Discussions!

### DISCUSSION THREAD

**Discussion by Kushal997-das** [Kushal997-das/Hacktoberfest-2021#1] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lakhankumawat** [comment]
Hi @Kushal997-das  can you please add my project  [LearnCpp](https://github.com/Lakhankumawat/LearnCPP) in project guidance section of [README](https://github.com/Kushal997-das/Hacktoberfest_2021/blob/master/readme.md) it would be grateful for me .

I could create a pull request for this but please don't count it towards hacktoberfest 2021 accepted as it won't be a quality contribution.
Thanks

**Comment by Kushal997-das** [comment, marked as answer]
@Lakhankumawat This is not a project ,  Kind of source it is and @Lakhankumawat `Project-Guidance` only meant for taking project . You can read the contribution.md file then you get to know . I hope you understood . Thanks :)

**Reply by Lakhankumawat** [reply]
It is not a project it is true but the purpose of mentioning in readme was just to get a guidance na 😅 , that's what i am saying , So it can be a good guidance specially for beginners , because in my repo anyone can contribute freely but in projects and heavy websites a simple bug can stop development cycle , mine repo is better at guidance , hope you got it ! After all choice is all yours 😁

**Reply by Kushal997-das** [reply]
@Lakhankumawat First of all if i gonna add your Project then this will harm for me because hacktoberfest2021 this time they are very strict towards contribution . Even this repo has been excluded from hacktoberfest2021 and i think your too . So, i don't want to make my project-guidance as a spam project .More over this project-guidance itself helps the people to find their respective project . I hope you understand . Thanks @Lakhankumawat

**Reply by Lakhankumawat** [reply]
okay no issues 👱‍♂️
